packages/@sanity/base/src/change-indicators/helpers/getElementGeometry.ts

```diff
@@ -0,0 +1,10 @@
+import getOffsetsTo from './getOffsetsTo'
+import {Rect} from '../overlay/types'
+
+export function getElementGeometry(element, parent): {rect: Rect; bounds: Rect} {
```

Comment:
Are the arguments missing types or is typescript okay with it? :)

```suggestion
export function getElementGeometry(element: HTMLElement, parent: HTMLElement): {rect: Rect; bounds: Rect} {
```

Reply:
Great point! Eslint actually complains here, so will fix :)

The reason typescript is happy is because strict mode is off in this project (due to legacy reasons). I'll check how much work it is to enable it only for this subfolder.

Reply:
Oh I see! I didn't know that :)